third_party/move/move-compiler/transactional-tests/tests/control_flow/function_call_mut_arg_twice.move

```diff
@@ -0,0 +1,18 @@
+//# publish
```

Comment:
Looks like many non-for-loop related tests (like this one) were also added. I am assuming this is intentional.

Reply:
I was making /control_flow/ the same for V1 and V2.

Reply:
But I've also backpedaled and removed a test that doesn't yield expected warnings on -v2 now.